src/contribute/portability.md

```diff
@@ -174,6 +174,18 @@ their respective components.
 
 ### Status
 
-MVP in progress. See the
+MVP complete (for Flink runner). See the
 [Portability support table](https://docs.google.com/spreadsheets/d/1KDa_FGn1ShjomGd-UUDOhuh2q73de2tPz6BqHpzqvNI/edit?usp=sharing)
 for details.
+
+#### Running Python wordcount on Flink
+
+The Flink runner is currently the only runner to support portable pipeline execution.
+To run a basic Python wordcount (in batch mode):
+
+1. Run once to build the SDK harness container: `./gradlew -p sdks/python/container
+docker`
+2. Start the Flink portable JobService endpoint: `./gradlew :beam-runners-flink_2.11-job-server:runShadow`
+3. Submit the wordcount pipeline to above endpoint: `./gradlew :beam-sdks-python:portableWordCount`
```

Comment:
We can also add instructions to schedule it on a local flink cluster.

`./gradlew :beam-runners-flink_2.11-job-server:shadowJar` and then run the jar with appropriate arguments including endpoint for FlinkJobserver